manifests/server/redhat.pp

```diff
@@ -6,11 +6,24 @@
 #
 class nfs::server::redhat inherits nfs::client::redhat {
 
-  service{ 'nfs':
+  $servicename = $::operatingsystemmajrelease ? {
+    '7'     =>  'nfs-server',
+    default => 'nfs'
+
+  }
+
+  service{ $servicename:
     ensure    => running,
     enable    => true,
     hasstatus => true,
     require   => Package[ 'nfs-utils' ],
   }
 
+  @concat {'/etc/exports':
```

Comment:
same thing happens in Debian form. On nfs::Server this fails without @concat for exports in RHEL 7.